core/src/mindustry/entities/comp/WeaponsComp.java

```diff
@@ -148,8 +148,13 @@ public void update(){
             if(weapon.rotate && (mount.rotate || mount.shoot) && can){
                 float axisX = this.x + Angles.trnsx(this.rotation - 90,  weapon.x, weapon.y),
                     axisY = this.y + Angles.trnsy(this.rotation - 90,  weapon.x, weapon.y);
-
                 mount.targetRotation = Angles.angle(axisX, axisY, mount.aimX, mount.aimY) - this.rotation;
+                if(weapon.rotateLimit > 0){
+                    mount.rotation = Angles.moveToward(mount.rotation, Mathf.clamp(mount.targetRotation, -(weapon.rotateLimit), weapon.rotateLimit), weapon.rotateSpeed * Time.delta);
+                }
+                else{
+                    mount.rotation = Angles.moveToward(mount.rotation, mount.targetRotation, weapon.rotateSpeed * Time.delta);
+                }
```

Comment:
> Style. This should be 1 line.

```suggestion
                if(weapon.rotateLimit > 0) mount.rotation = Angles.moveToward(mount.rotation, Mathf.clamp(mount.targetRotation, -(weapon.rotateLimit), weapon.rotateLimit), weapon.rotateSpeed * Time.delta);
                else mount.rotation = Angles.moveToward(mount.rotation, mount.targetRotation, weapon.rotateSpeed * Time.delta);
```
(Does this work?)

Reply:
No, I only mean
```
}
else{
```
should be one line, like
```
}else{
```

Reply:
ah ok